.github/workflows/ccruntime_e2e_amd.yaml

```diff
@@ -0,0 +1,56 @@
+name: ccruntime e2e tests amd sev
+on:
+  workflow_call:
+
+permissions:
+  contents: read
+
+jobs:
+  e2e:
+    name: operator tests
+    strategy:
+      fail-fast: false
+      matrix:
+        runtimeclass: 
+          - "kata-qemu-sev"
+          - "kata-qemu-snp"
+        instance: 
+          - "coco-ci-amd-rome-001"
+          - "coco-ci-amd-milan-001"
```

Comment:
would be nice if we could use the label here instead. Adi said he'd try this.

Reply:
Keeping it consistent with Choi by using the instance instead of label.